lib/GetStream/StreamChat/Client.php

```diff
@@ -546,6 +546,28 @@ public function unFlagUser(string $targetId, array $options = null): StreamRespo
         return $this->post("moderation/unflag", $options);
     }
 
+    /** @link https://getstream.io/chat/docs/php/moderation/?language=php
+     * @throws StreamException
+     */
+    public function queryFlagReports(array $options): StreamResponse
+    {
+        $data = ["filter_conditions" => $options];
+        return $this->post("moderation/reports", $data);
+    }
+
+    /** @link https://getstream.io/chat/docs/php/moderation/?language=php
+     * @throws StreamException
+     */
+    public function reviewFlagReport(string $reportId, string $reviewResult, string $userId, array $details): StreamResponse
```

Comment:
don't we want to provide a better for response ?

Reply:
right now, it's kind of untyped like this everywhere